src/components/features/DeliveryGoodsInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, Typography } from '@mui/material'
+import { useTranslation } from 'react-i18next'
+
+interface GoodType {
+  title: string
+  value: string
+  quantity: number
+}
```

Comment:
dont we already have a type for this?

Reply:
No. 
![image](https://github.com/InterCarsBulgaria-TechLeadAcademy/warehouse.management.fe/assets/123353314/ee37c95f-d9e4-44f4-b463-7ea061e1bff8)